packages/package-store/src/storeController/createImportPackage.ts

```diff
@@ -59,19 +59,22 @@ function createAutoImporter () {
   ) {
     try {
       await clonePkg(to, opts)
+      packageImportMethodLogger.debug({ method: 'clone' })
```

Comment:
package import method should also be logged at each switch case of the createImportPackage function